server/segment_rpc.go

```diff
@@ -192,6 +192,16 @@ func (h *lphttp) ServeSegment(w http.ResponseWriter, r *http.Request) {
 			Url:    uri,
 			Pixels: res.TranscodeData.Segments[i].Pixels,
 		}
+		// Save perceptual hash if generated
+		if res.TranscodeData.Segments[i].PHash != nil {
+			pHashFile := name + ".phash"
```

Comment:
I'm guessing the extension doesn't matter here right? We're using `.phash` here, but we could just as well use `.bin` which ffmpeg uses?

Reply:
Yep file extensions are inconsequential for non-standard binary data. Chose `.phash` for the object store because it's more specific than the generic `.bin`.